.github/workflows/docs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: [3.7]
```

Comment:
does this have to be reverted? same for line 8

Reply:
no I guess, i'll bump it back up then